system/datatypes/text.php

```diff
@@ -20,6 +20,7 @@ function __construct($pageListNode, $args) {
 			registerCommandCallback('textRevert', Array($this, 'revert'));
 		}
 
+		/** @deprecated  */
```

Comment:
This should probably not go into this commit, the commit message does not mention this change. Also, why is this deprecated?

Reply:
You are right, this should go out, I'll fix it.